src/parser/toplevel.lisp

```diff
@@ -265,6 +276,22 @@
 (deftype toplevel-define-type-list ()
   '(satisfies toplevel-define-type-list-p))
 
+(defstruct (toplevel-define-alias
+            (:include toplevel-definition)
+            (:copier :nil))
```

Comment:
nil not :nil

Reply:
resolved in [ff0e7f7](https://github.com/coalton-lang/coalton/pull/1294/commits/ff0e7f71131c98c21d973ad71b31bf69b222b51d)